nitrite/src/main/java/org/dizitart/no2/util/ObjectUtils.java

```diff
@@ -63,6 +63,21 @@
         return type.getName();
     }
 
+    /**
+     * Generates the name of an {@link org.dizitart.no2.objects.ObjectRepository}
+     * with an unique key identifier.
+     *
+     * @param <T>  the type parameter
+     * @param key  the key identifier
+     * @param type the type of object stored in the repository
+     * @return the name of the object repository.
+     */
+    public static <T> String findObjectStoreName(String key, Class<T> type) {
+        notNull(type, errorMessage("key can not be null", VE_OBJ_STORE_NULL_TYPE));
```

Comment:
Please create a new error code `VE_OBJ_STORE_NULL_KEY` to differentiate it from the `VE_OBJ_STORE_NULL_TYPE`